src/__tests__/Search.test.js

```diff
@@ -0,0 +1,11 @@
+import { fireEvent, getByRole, render, screen } from "@testing-library/react";
```

Comment:
You can remove most of these imports here as you are not currently using them in the tests:
```
import { render } from "@testing-library/react";
```